components/mdx/Spoiler/spoiler.module.scss

```diff
@@ -0,0 +1,3 @@
+.content {
+  margin-left: 0.9rem;
```

Comment:
This should be `1rem` instead.

Reply:
`1rem` goes a bit away from the letter "A", I wanted it to be close to perfection.